src/main/java/nl/workingtalent/bookrental/dto/ReservationSendDto.java

```diff
@@ -0,0 +1,53 @@
+package nl.workingtalent.bookrental.dto;
+
+import nl.workingtalent.bookrental.model.Book;
+import nl.workingtalent.bookrental.model.User;
+
+public class ReservationSendDto {
```

Comment:
Gingen we toch send & request dto's opsplitsen? Ik dacht dat we vanwege het werk zouden blijven bij 1 versie dto

Reply:
Als het werkt, is het alvast mooi meegenomen 👍